batch7crud-roll9/src/main/java/com/lftechnology/batch7crud/services/StudentService.java

```diff
@@ -0,0 +1,39 @@
+package com.lftechnology.batch7crud.services;
+
+import com.lftechnology.batch7crud.dao.StudentDAO;
+import com.lftechnology.batch7crud.entity.Student;
+import com.lftechnology.batch7crud.exception.DataException;
+
+import java.util.ArrayList;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'java.util.ArrayList'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)